src/lib/ibcclient.ts

```diff
@@ -210,6 +212,7 @@ export class IbcClient {
     IbcExtension;
   public readonly tm: TendermintClient;
   public readonly senderAddress: string;
+  public readonly logger?: Logger;
```

Comment:
Why not make it required?

Reply:
Mostly because I would have had to update many more irrelevant lines haha. Not sure if there's a use case for no logger though?

Reply:
I would set it to a default `NoopLogger` implementation in the IbcClient constructor if none was provided. This means, we can always trust it to exist and call it (which will lave many more lines later), but it does nothing by default.

Reply:
Cool, I'll add one.